pom.xml

```diff
@@ -1257,6 +1243,17 @@ under the License.
 		<pluginManagement>
 			<plugins>
 
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-release-plugin</artifactId>
```

Comment:
We were actually not using this before, it was just a leftover in there. It's not even required for our release process, i.e. in the release profile.

However, the plugin can be used to do the stuff that we do in scripts and manually. For example , I can do
```
mvn release:branch \
    -DbranchName=${BRANCH_NAME} \
    -DdevelopmentVersion=${DEVELOPMENT_VERSION}
```
to update the version on master and fork of the branch for a newly released version.

Then I could use
```
mvn release:prepare \
    -Dresume=false \
    -DreleaseVersion=${VERSION} \
    -Dtag=${TAG} \
    -DupdateWorkingCopyVersions=false
```
and
```
mvn release:perform
```
to build the actual release, create the source release, upload the (signed) jars to sonatype and create the release tag.

(I was experimenting with this when preparing for the next release and our release infrastructure is very _interesting_.)